Classes/Controller/PlaceController.php

```diff
@@ -78,30 +78,21 @@ public function showAction(): ResponseInterface
 
         if (!empty($place)) {
             unset($place['photo']);
-            $place['photos'] = [];
-            $fileRepository = GeneralUtility::makeInstance(FileRepository::class);
 
             $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
                 ->getQueryBuilderForTable('sys_file_reference');
-            $fileUids = $queryBuilder
-                ->select('uid_local')
+            $place['photos'] = $queryBuilder
+                ->select('uid')
                 ->from('sys_file_reference')
                 ->where(
-                    $queryBuilder->expr()->eq('uid_foreign', $queryBuilder->createNamedParameter($place['uid'], Connection::PARAM_INT)),
+                    $queryBuilder->expr()->eq('uid_foreign', $queryBuilder->createNamedParameter($place['uid'], \PDO::PARAM_INT)),
                     $queryBuilder->expr()->eq('tablenames', $queryBuilder->quote('tx_theodia_place')),
                     $queryBuilder->expr()->eq('fieldname', $queryBuilder->quote('photo'))
                 )
                 ->orderBy('sorting_foreign')
-                ->executeQuery()
+                ->execute()
                 ->fetchFirstColumn();
 
-            foreach ($fileUids as $fileUid) {
-                $imageFile = $fileRepository->findByUid($fileUid);
-                if ($imageFile !== null) {
-                    $place['photos'][] = $imageFile;
-                }
-            }
-
```

Comment:
I do get that you now return the `sys_file_reference` record, and thus you get cropping but previously you had an `File` object, now you only have a UID, possibly this should better be transformed into some "FileReference" object as it is common with Extbase, this would make the Fluid much more clean.